include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -183,13 +191,22 @@ namespace aspect
 
 
         private:
+
+          /**
+           * Reference strain rate for the first time step.
```

Comment:
Suggestion: `Reference strain rate for the first non-linear iteration in the first time step`